src/components/Book.jsx

```diff
@@ -1,19 +1,22 @@
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 import style from '../style/Book.module.css';
 
 function Book({ book }) {
+  const dispatch = useDispatch();
   return (
     <div className={style.bookContainer}>
       <div className={style.bookDetails}>
         <div className={style.book}>
-          <h4>{book.nature}</h4>
+          <h4>{book.category}</h4>
           <h3>{book.title}</h3>
           <p>{book.author}</p>
         </div>
         <ul className={style.booOptions}>
-          <li>Comments</li>
-          <li>Remove</li>
-          <li>Edit</li>
+          <li><a href="#coment">Comments</a></li>
+          <li><a href="#remove" onClick={() => { dispatch(removeBook(book)); }}>Remove</a></li>
```

Comment:
- [ ] Could you please think about moving the remove book logic to a different component? By doing this, the code will become more organized and easier to read. 

   ![image](https://github.com/Graciano1997/book_store/assets/107952724/de671133-8da0-4045-876e-f6538485ce9d)

Reply:
@taldr27  it is hard but it improves me 🙏 🙏 🙏 🙏 very much!